api_tests/tests/lightning_routes.ts

```diff
@@ -18,22 +18,23 @@ describe("Lightning routes tests", () => {
             const bodies = (await SwapFactory.newSwap(alice, bob, true))
                 .hanEthereumEtherHalightLightningBitcoin;
 
-            const aliceResponse = alice.cnd.createHanEthereumEtherHalightLightningBitcoin(
```

Comment:
The reason this was split was so the test follows the "arrange-act-assert" structure :)

Reply:
Yes but the test is unstable. For some reason it fails for the exact reason it should fail but jest does not catch the promise rejection. See commit comment.

Reply:
Yeah I noticed that. Does this actually fix it? That is so weird ...

Reply:
I did ~5 full runs since then and the issue did not come back...

Reply:
Hmm, well spotted then :)
I wonder if we could reproduce that somehow and report a bug!